client/src/main/java/org/apache/rocketmq/client/impl/consumer/DefaultLitePullConsumerImpl.java

```diff
@@ -691,7 +691,7 @@ public void run() {
 
                 ProcessQueue processQueue = assignedMessageQueue.getProcessQueue(messageQueue);
 
-                if (processQueue == null && processQueue.isDropped()) {
+                if (processQueue != null && processQueue.isDropped()) {
```

Comment:
assignedMessageQueue.getProcessQueue(messageQueue) may return null. So it is necessary to check if processQueue is null to prevent NPE

Reply:
ok,i check both with (processQueue is null) or (processQueue not null and isDropped)